ironfish/src/wallet/wallet.ts

```diff
@@ -1599,6 +1617,31 @@ export class Wallet {
     return this.getAccountByName(name) !== null
   }
 
+  async getIdentityName(identity: string): Promise<string | undefined> {
```

Comment:
Nit: `getMultisigIdentityName`?

Also, I always thought that this "name" was the name of a (partial) account, am I wrong? Is this thing totally separated from the account?

Reply:
There's no explicit linking between the two, and there's no guarantee in the code that they are the same. I think that was the intention, but we discovered via live testing it can be broken fairly easily. Will need some sort of join index/store between the two I think.